src/regions-mixin.js

```diff
@@ -11,7 +11,7 @@ export default {
   _initRegions: function(options) {
 
     // init regions hash
-    this.regions =  this.regions || {};
+    this.regions =  _.clone(_.result(this, 'regions', {}));
```

Comment:
I think this is actually just ensuring the hash exists.. we shouldn't do the result here.  `addRegions` will do that